lcd_hd44780.lua

```diff
@@ -159,4 +163,15 @@ function lcd_hd44780:has_two_e()
     end   
 end
 
+function lcd_hd44780:setCustomChar(position, pattern)
+	drv:command(bit.bor(0x40, bit.lshift(position, 3)), lcd_hd44780.detect_e(self))
+
+	for key,value in pairs(pattern) do
+		drv:command(pattern[key], lcd_hd44780.detect_e(self), true)
```

Comment:
maybe it would be better to use write instead of command, we won't need another argument in **:command** function or even set RS to high before **for** and use **_write8**?